### PR TITLE
run TestKitchen build on Travis-CI

### PR DESCRIPTION
**Pull request by jbabe**
🐘 

- runs http://kitchen.ci tests via Travis-CI using an Ubuntu 14.04 Docker container image
- install busser to run serverspec tests on container
- adds a couple basic tests (more to come).
- update logrotate paths to match more generic /var/log pattern

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,29 +1,20 @@
...
-  apt:
-    packages:
-    - python-pip
-
+language: ruby
+sudo: required
+rvm: 2.3.0
+cache: bundler
+env:
+  - distro: ubuntu1404
```
Comment (MathieuGilbert) [inline .travis.yml]: Are we using 16.04 now?

File: .kitchen.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+provisioner:
+  name: ansible_playbook
+  roles_path: ./
+  # so that everything under this folder gets install in /tmp/kitchen/roles/loggly_tls instead of /tmp/kitchen/roles
+  role_name: loggly_tls
+  hosts: test-kitchen
+
+platforms:
+  - name: ubuntu-14.04
```
Comment (MathieuGilbert) [inline .kitchen.yml]: 16.04?

**Comment by MathieuGilbert** [comment]
💩

File: .travis.yml (hunk 1 above)
Comment (jbabe) [inline .travis.yml]: We are. I'll add it too so this builds on both

**Comment by jbabe** [comment]
Will need to update this with master. I may try another branch that just uses the container that the tests are running on on TravisCI and not worry about test kitchen.

**Pull request closed without merging**